tools/eslint-rules/utilities/member-expression/get-member-expressions-from-node.ts

```diff
@@ -0,0 +1,627 @@
+import { AST_NODE_TYPES, TSESTree } from '@typescript-eslint/utils';
```

Comment:
are all of these overloads actually being used? xD

Reply:
"yes"